src/styles/components/_modal.scss

```diff
@@ -1,88 +1,74 @@
-.bsds-modal
-{
+.bsds-modal {
+  // NOTE: This is a one off size. We can keep an eye out as we build more modal/panel type components to see if this is a dimension we want to flip to a variable.
+  min-width: min(37.5rem, 100vw);
   box-shadow: $box-shadow-primary;
   border: 0;
   padding: space(4x);
   background: $neutral-100;
   max-width: 90vw;
-  min-width: min(37.5rem, 100vw); // NOTE: This is a one off size. We can keep an eye out as we build more modal/panel type components to see if this is a dimension we want to flip to a variable.
 
-  &::backdrop
-  {
+  &::backdrop {
     background: rgba($neutral-00, 0.6);
   }
 
-
-  .bsds-modal-header
-  {
+  .bsds-modal-header {
     display: grid;
     grid-template-columns: 1fr auto;
     grid-template-areas:
-      "logo close"
-      "title close";
+      'logo close'
+      'title close';
     align-items: start;
     gap: space(3x);
   }
 
-  .bsds-modal-logo
-  {
-    grid-area: logo;
-    margin-top: 0;
+  .bsds-modal-logo {
     width: auto;
     height: space(6x);
+    margin-top: 0;
+    grid-area: logo;
```

Comment:
feels strange grid-area is this low. let's double check the property order list to make sure our grid properties are grouped near display/flex.